docs/source/AdministratorGuide/ServerInstallations/environment_variable_configuration.rst

```diff
@@ -31,6 +31,9 @@ DIRAC_USE_JSON_ENCODE
 DIRAC_USE_M2CRYPTO
   If anything else than ``true`` or ``yes`` (default) DIRAC will revert back to using pyGSI instead of m2crypto for handling certificates, proxies, etc.
 
+DIRAC_M2CRYPTO_DEBUG
```

Comment:
There is already a couple `DIRAC_DEBUG_*` variables. Use the same for this one?

Can you remove the duplicate entry for DIRAC_USE_M2CRYPTO, preferably the one that is less close to its alphabetic order position :)

Reply:
done, thanks